.github/workflows/test-google-cloud-platform-install.yaml

```diff
@@ -0,0 +1,204 @@
+#
+# This is an e2e test to deploy PostHog on Google Cloud Platform using Helm.
+#
+# TODO:
+# - run k8s spec test
+# - run action only when necessary
+#
+name: e2e - Google Cloud Platform (install)
+
+on: push
```

Comment:
can we add `workflow-dispatch` to all of these too, just so we can trigger them from actions if we want to